.babelrc

```diff
@@ -1,5 +1,5 @@
-{
-
+  {
```

Comment:
Lose these changes. It's not needed

Reply:
For this I rebase from develop, thats how it looks but. I dont know why it keeps saying it is me who edited those lines

Reply:
and when I try to rebase now, it is giving me that branch is update